tests/test_pip.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from testinfra.utils.ansible_runner import AnsibleRunner
+
+
+@pytest.fixture()
+def AnsibleDefaults(Ansible):
+    """ Load default variables into dictionary.
+
+    Args:
+        Ansible - Requires the ansible connection backend.
+    """
+    return Ansible("include_vars", "./defaults/main.yml")["ansible_facts"]
+
+
+@pytest.fixture()
+def AnsibleFacts(Ansible):
+    """ Load ansible facts into a dictionary.
+
+    Args:
+        Ansible - Requires ansible connection backend.
+    """
+
+    return Ansible("setup")["ansible_facts"]
+
+
+def test_pip_exists(Package, AnsibleDefaults, AnsibleFacts):
+    """ Ensure pip is installed, with the proper version for the OS.
+
+    Args:
+        Package - Module to determine package install status and version
+        AnsibleDefaults - Module to determine pip install version
+        AnsibleFacts - Used to determine OS Distribution Version
+    """
+    pip = Package("python-pip")
+    os_version = AnsibleFacts["ansible_distribution_version"]
+    if float(os_version) >= 16.04:
```

Comment:
I'm not sure that this approach is going to be sustainable down the road. I think that we should leave the default version as something that works with Trusty and leave it up to the person running the tests to update it for other distributions.

I realize that this'll prevent us from being able to test both platforms back-to-back, but keeping these package versions up-to-date for all platforms is going to be a burden.